posthog/tasks/alerts/test/test_alert_checks.py

```diff
@@ -419,3 +419,132 @@ def test_alert_is_checked_on_weekday_when_skip_weekends_is_true(
 
         checks = AlertCheck.objects.filter(alert_configuration=self.alert["id"])
         assert len(checks) == 1
+
+    def test_alert_triggered_for_single_formula(
```

Comment:
Do we support legacy `"formula"` as well? Most of the existing insight still follow this format.